etc/org.opencastproject.workingfilerepository.impl.WorkingFileRepository.cfg

```diff
@@ -0,0 +1,21 @@
+# Filename pattern
+#
+# Opencast will filter out dangerous pattern from file names in order to make these compatible with different operating
+# and file systems and to prevent local security risks like path traversal. For example, the default conversion would
+# look like this:
+#
+#   original:  -problematic-file/../name&&&%.ext
+#   converted: _problematic-file____name____.ext
+#
+# This option allows you to control which characters are not allowed. All parts of the file name matched by the
+# configured pattern will be replaced by `_`. The replacement is applied separately to the base file name and the
+# extension. In any case, both base name and extension are limited to 255 characters each.
+#
+# WARNING: This setting is dangerous. It lets you configure Opencast in a way that poses a security risk to your system
+#          by e.g. allowing for path traversal attacks.
+# BETA:    For a long time, this was a very restricted, fixed value. Consider this to be an unstable configuration which
+#          might lead to problems since some component somewhere assumes the old restricted behavior. Test your system
+#          very carefully after changing this. Feedback about problems and successful changes are welcome.
+#
+# Default: (^\\W|[^\\w-_.])
+#filename.pattern = (^\\W|[^\\w-_.])
```

Comment:
The underscore `_` is part of `\w`, according to [Oracle](https://docs.oracle.com/javase/tutorial/essential/regex/pre_char_classes.html).